File: data/src/main/java/com/github/gmetal/data/entity/CreatedByEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class CreatedByEntity {

    @JsonField(name = "id")
    public Integer id;
    @JsonField(name = "name")
    public String name;
    @JsonField(name = "profile_path")
    public String profilePath;
}

File: data/src/main/java/com/github/gmetal/data/entity/CreatedByEntity.kt
package com.github.gmetal.data.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class CreatedByEntity {

    @SerialName("id")
    var id: Int? = null
    @SerialName("name")
    var name: String? = null
    @SerialName("profile_path")
    var profilePath: String? = null
}
